rubocop-inflector.gemspec

```diff
@@ -7,8 +7,8 @@ require 'rubocop/inflector/version'
 Gem::Specification.new do |spec|
   spec.name          = 'rubocop-inflector'
   spec.version       = RuboCop::Inflector::VERSION
-  spec.authors       = ['Takumasa Ochi']
-  spec.email         = ['aeroastro007@gmail.com']
+  spec.authors       = ['Takumasa Ochi', 'Dylan Rainwater']
+  spec.email         = ['aeroastro007@gmail.com', 'support@splitwise.com']
```

Comment:
Authors and emails are a little bit different notion from contributors.
So, can you omit this changes in this Pull Request.

Later, I can add your contribution to the changelog as follows.

```markdown
* Widen required_ruby_version to be compatible with Ruby 3 [Dylan Rainwater](@dylanrainwater)
```

https://github.com/aeroastro/rubocop-inflector/blob/2ab27d9682588b12f39d07d1b0134ecde57a34c5/CHANGELOG.md?plain=1#L9-L10